module-4/backend/week_one.md

```diff
@@ -1,20 +1,32 @@
-## Week One - Module 4 Recap
+## week_one.md
 
-Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week. 
+### What's the most useful thing you learned from completing the intermission week work?
+*Java script basics*
+### What are some tools to help debug JavaScript code?
+*console.log*
+*eval(pry.it)*
+### What are some tools you need in order to unit test your JavaScript?
+*install chai and mocha*
+### What is the syntax for invoking a function?
+*myFunction()*
+### What's the difference between == and === in JavaScript?
+*By using == you check if something is equal to something else. This is not strict and by using === its also strict*
+### What's the difference between asynchronous and synchronous JavaScript?
+*asynchronicity is about how to pack up code to execute upon some later event*
+### How do we setup a route when creating an API with Node and Express?
+*create instance of express named app*
+*then call HTTP equest method*
+*then specify path*
+*and then create function*
+### What do we use Knex for?
+*we use it to work (build) database*
+### What's npm and what do we use it for?
+*package manager for JavaScript. It allows users to consume and distribute JavaScript modules*
+## Review
 
-Note: When you're done, submit a PR. 
-
-1. What's the most useful thing you learned from completing the intermission week work?
-2. What are some tools to help debug JavaScript code?
-3. What are some tools you need in order to unit test your JavaScript?
-4. What is the syntax for invoking a function?
-5. What's the difference between `==` and `===` in JavaScript?
-6. What's the difference between asynchronous and synchronous JavaScript? 
-7. How do we setup a route when creating an API with Node and Express?
-8. What do we use Knex for?
-9. What's `npm` and what do we use it for?
-
-#### Review  
-10. What's the MVC design pattern? Describe each part of MVC?
-11. What is AJAX? What are some benefits of using AJAX?
-12. What's a background worker? When would we want to use a background worker?
+### What's the MVC design pattern? Describe each part of MVC?
+*Model, View, Controller*
+### What is AJAX? What are some benefits of using AJAX?
+*asynchronous JavaScript and XML. Ajax calls can help decrease website load*
+### What's a background worker? When would we want to use a background worker?
+*Executes an operation on a separate thread. We use it when we need to continue to work with the page while web worker runs in the background*
```

Comment:
Nice! I'm not sure if it's necessarily or always a different thread exactly. A lot of times the job is just delayed to execute at another time.